src/Command/VotingPlatform/ConfigureCommand.php

```diff
@@ -106,6 +109,44 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return 0;
     }
 
+    private function configureCommitteeSupervisorElections(Designation $designation): void
+    {
+        $offset = 0;
+
+        $now = new \DateTime();
+
+        while ($committeeElections = $this->committeeElectionRepository->findAllByDesignation($designation, $offset)) {
+            foreach ($committeeElections as $committeeElection) {
+                $committee = $committeeElection->getCommittee();
+
+                if (!$election = $this->electionRepository->hasElectionForCommittee($committee, $designation)) {
+                    $election = $this->createNewElection($designation, $electionEntity = new ElectionEntity());
+                    $electionEntity->setCommittee($committee);
+                    $this->configureNewElectionForCommitteeSupervisor($election);
+                }
+
+                if (
+                    $designation->getVoteStartDate() < $now->modify('+10 minutes')
+                    && !$this->committeeElectionHasCandidates($committee, $designation)
```

Comment:
```suggestion
                    && !$committeeElection->countConfirmedCandidacies()
```

Reply:
I think we can replace the method `committeeElectionHasCandidates` by `&& !$committeeElection->countConfirmedCandidacies()`